src/main/kotlin/org/jetbrains/teamcity/rest/api.kt

```diff
@@ -181,3 +183,15 @@ interface TriggeredInfo {
     val user: User?
     val build: Build?
 }
+
+interface FinishBuildTrigger {
+    val initiatedBuildConfiguration: BuildConfigurationId
+    val afterSuccessfulBuildOnly: Boolean
+    val branchFilter: List<BranchRule>
+}
+
+interface BranchRule {
+    val rule: String
+    val include: Boolean
+    val name: String
```

Comment:
`name` of what? The interface is named BranchRule, so it isn't evident if it is a name of a branch or name of a rule. I would either add a javadoc or rename the property to `branchName`.